### PR TITLE
Delete unused manipulations

### PR DESCRIPTION
**Pull request by yafbu**
Deletes in RBT no longer used manipulations

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

**Review by ben-Draeger — APPROVED** [review]

**Comment by antipovi** [comment]
should we add the deleted manipulation to changelog?

File: src/t2iapi/alert/alert_requests.proto [hunk 1]
```diff
@@ -52,12 +52,4 @@ message AlertConditionEscalationRequest {
   AlertConditionEscalationProcess escalation_process = 2;
 }
 
-/*
-This message is used as a request to trigger a MDIB change, which causes the AlertCondition corresponding to the given
-handle to change its Presence to the given value.
- */
-message SetAlertConditionAndReturnCauseRequest {
```
Comment (antipovi) [inline src/t2iapi/alert/alert_requests.proto:59]: should we add the deleted manipulation to changelog?

Comment (ben-Draeger) [reply src/t2iapi/alert/alert_requests.proto:59]: Yes, why not?

Comment (antipovi) [reply src/t2iapi/alert/alert_requests.proto:59]: @yafbu please add the changes to changelog

**Pull request merged**